tests/XPath2.Tests/XPathNavigatorTests.cs

```diff
@@ -186,6 +186,52 @@ public void XPath2Evaluate_Arithmetic_With_DecimalConversion()
             Assert.Equal(true, equal2);
         }
 
+        [Fact]
```

Comment:
There are some merge conflicts on the test file. Can you fix this?

Reply:
I am kind of lost of what the issue is, the GitHub page shows line 189 of XPathNavigatorTest.cs. I will need to look tomorrow to see if I understand which conflict there is.

Reply:
The best solution is to just copy the new tests in the existing test class from master.

Reply:
I currently don't get it, did you mean to say "to copy the new tests **into** the existing test class from master"? 

"to just copy the new tests in the existing test class from master" sounds as my branch lacks tests that are in the master branch.

Reply:
I copied two tests that were at the end of the file in your master branch but not in my branch. https://github.com/martin-honnen/XPath2.Net/commit/2a8d3d88c05b5b7cd626c467294a82059eedb422